templates/internal/appName/k8s/activity.go.tpl

```diff
@@ -66,8 +67,9 @@ func (s *KubernetesService) Run(ctx context.Context) error { //nolint: funlen,ll
 	ctrl.SetLogger(logadapters.NewLogrLogger(ctx))
 
 	s.registerSchemes()
-
-	options := ctrl.Options{
+  
+    // manager options
+    mgrOptions := ctrl.Options{
```

Comment:
can we use the whole word? `manager` was not the first thing i thought of

Reply:
Ok, changed